docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+# VmAnomaly Quickstart
+
+### Prerequisites
+- In this tutorial it is assumed that you have VictoriaMetrics already, and you are familiar with VictoriaMetrics ecosystem:
+  -  [victoriametrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html)
+  
+- It is assumed that you are familiar with Grafana and Docker
+## What is VmAnomaly?
+*VictoriaMetrics Anomaly Detection* is a service that continuously scans VictoriaMetrics time series and detects unexpected changes within data patterns in real-time. It does so by utilizing user-configurable machine learning models.
+
+All the service parameters (model, schedule, input-output) are defined in a config file.
+
+Single config file supports only one model, but it's totally OK to run multiple vmanomaly processes in parallel, each using its own config.
+
+VmAnomaly periodically queries user-specified metrics, computes an **anomaly score** for them, based on how well they fit a predicted distribution, taking into account periodical data patterns with trends and/or seasonality, and pushes back the computed "anomaly score" to VictoriaMetrics. Then, users can enable alerting rules based on the "anomaly score" with VmAlert.
+
+
+## What is VmAlert?
+VMalert is an alerting tool for VictoriaMetrics. It executes a list of the given alerting or recording rules against configured `-datasource.url` compatible with Prometheus HTTP API.
+
+Alerting rules allow you to define alert conditions based on Prometheus expression language expressions and to send notifications about firing alerts to an external service. Whenever the alert expression results in one or more vector elements at a given point in time, the alert counts as active for these elements' label sets.
```

Comment:
```suggestion
[Alerting rules](https://docs.victoriametrics.com/vmalert.html#alerting-rules) allow you to define conditions that, when met, will notify the user. The alerting condition is defined in a form of a query expression via [MetricsQL query language](https://docs.victoriametrics.com/MetricsQL.html). For example, the expression `cpu_usage_percent > 90` will notify a user when the current CPU usage will exceed 90%.
```

Reply:
rephrased the sentence, but changed the example, as cpu_usage_percent is going to confuse a user.
`For example, in our case, the expression `anomaly_score > 1.0` will notify a user when the calculated anomaly score exceeds a threshold of 1. `